### PR TITLE
components for fun and profit

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by alicoding** [comment]
This patch is huge

**Comment by ScottDowne** [comment]
Yeah but once it's done, sequential.jsx can be deleted, and likely shared.less too.

Oh right, this is a WIP. I still need to do a bunch of things...

File: components/address-input.jsx [hunk 1]
```diff
@@ -0,0 +1,628 @@
+var React = require('react');
```
Comment (alicoding) [inline components/address-input.jsx]: Import?

File: components/amount-buttons.jsx [hunk 2]
```diff
@@ -4,17 +4,68 @@ var AmountButtons = React.createClass({
...
+    this.setAmount();
+  },
+  setAmount: function() {
+    this.setState({
+      valid: true
+    });
+    this.props.onChange(this.props.name, this);
+  },
+  otherRadioClick: function() {
+    $("#amount-other-input").focus();
```
Comment (alicoding) [inline components/amount-buttons.jsx]: Don't use jquery and use either native or react built-in query selector

**Comment by alicoding** [comment]
rebase and fix that jquery stuff

**Comment by ScottDowne** [comment]
@alicoding rebased and ready for r again

**Pull request merged**